extensions/apps/settings/src/components/pages/notifications-option.tsx

```diff
@@ -68,39 +89,77 @@ const NotificationsOption: React.FC = () => {
     );
   }
 
+  const handleEnableNotifications = async () => {
+    setLoading(true);
+    let result: keyof typeof TOAST_TEXTS;
+    try {
+      await sdk.services.common.notification.initialize({ readonly: false });
+      result = 'success';
+      setNotificationsEnabled(true);
+    } catch (error) {
+      result = 'error';
+      setNotificationsEnabled(false);
+    } finally {
+      setLoading(false);
+    }
+
+    _uiEvents.current.next({
+      event: NotificationEvents.ShowNotification,
+      data: {
+        type: result == 'success' ? NotificationTypes.Success : NotificationTypes.Error,
+        title: TOAST_TEXTS[result].title,
+        description: TOAST_TEXTS[result].description,
+      },
+    });
+  };
+
   return (
     <PageLayout title={t('Notifications Settings')}>
-      <Stack padding="p-4">
-        {notificationsSettingsItems.map((item: ISettingsItem, idx: number) => {
-          const baseStyle = `flex py-4 justify-between items-center ${
-            idx !== notificationsSettingsItems.length - 1
-              ? 'border(b-1 solid grey8 dark:grey5)'
-              : 'border-none'
-          }`;
+      {!notificationsEnabled && (
+        <Stack padding="p-4">
+          <NotificationSettingsCard
```

Comment:
Btw you can introduce some prop named like `noWrapperCard` so that inside `NotificationSettingsCard` you can use the flag to decide whether to wrap elements with `Card` component or not.

Reply:
Good point, updated.